components/Footer.js

```diff
@@ -1,7 +1,140 @@
-import FacebookIcon from '@material-ui/icons/Facebook'
-import InstagramIcon from '@material-ui/icons/Instagram'
-import LinkedInIcon from '@material-ui/icons/LinkedIn'
-import RedditIcon from '@material-ui/icons/Reddit'
-import TwitterIcon from '@material-ui/icons/Twitter'
+import * as React from 'react'
+import AppBar from '@mui/material/AppBar'
+import Box from '@mui/material/Box'
+import Toolbar from '@mui/material/Toolbar'
+import Typography from '@mui/material/Typography'
+import Button from '@mui/material/Button'
 import Link from 'next/link'
-import Button from 'react-bootstrap/Button'
+import YouTubeIcon from '@mui/icons-material/YouTube'
+import InstagramIcon from '@mui/icons-material/Instagram'
+import TwitterIcon from '@mui/icons-material/Twitter'
+import NotificationsIcon from '@mui/icons-material/Notifications'
+import FacebookIcon from '@mui/icons-material/Facebook'
+import LinkedInIcon from '@mui/icons-material/LinkedIn'
+import useCheckMobileScreen from '../utils/useCheckMobileScreen'
+import { makeStyles } from '@mui/styles'
+
+const useStyles = makeStyles({
```

Comment:
`makeStyles` is [deprecated in MUI v5](https://mui.com/styles/basics/) and won't work with react strict mode or react 18. But it should be pretty easy to refactor with `emotion` and the `sx prop`